### PR TITLE
Core2 compilation fail fixed when enabling LVGL

### PR DESCRIPTION
**Pull request by Jason2866**
## Checklist:
  - [x] The pull request is done against the latest development branch
  - [x] Only relevant files were touched
  - [x] Only one feature/fix was added per PR and the code change compiles without warnings
  - [x] The code change is tested and works on Tasmota core ESP8266 V.2.7.4.9
  - [x] The code change is tested and works with core ESP32 V.1.0.6
  - [x] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

_NOTE: The code change must pass CI tests. **Your PR cannot be merged unless tests pass**_

### CODE REVIEW

File: tasmota/displaydesc/M5stack_core2_display.ini [hunk 1]
```diff
@@ -34,6 +34,5 @@ E1,0F,00,0E,14,03,11,07,31,C1,48,08,0F,0C,31,36,0F
 :2,C8,00,00,02
 :3,68,00,00,03
 :i,21,20
-:TI2,38,22,21
-:B,60,0
```
Comment (s-hadinger) [inline tasmota/displaydesc/M5stack_core2_display.ini:38]: Why did you remove this line? It's better to keep it.

**Comment by Jason2866** [comment]
changed again.
